backend/src/model/route_van.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import ForeignKeyConstraint
+from sqlalchemy.orm import Mapped, mapped_column
+from src.db import Base
+
+
+class RouteVanModel(Base):
+    __tablename__ = "route_vans"
```

Comment:
Looking in alembic, I don't see a table named route_vans.

Reply:
Seems like I was working with an older copy of the alembic tables

Reply:
Why'd you mark this as resolved? Something definitely needs to change here.